draft-dss-star.md

```diff
@@ -101,200 +101,567 @@ In this document we describe one such system, namely Distributed Secret Sharing
+The Randomness Server then computes `randomness_response = SerializeElement(evaluated_element)` and
+sends it to the client using the content type "message/star-randomness-response". An example
+response is below.
 
-## Protocol Leakage
+~~~
+:status = 200
+content-type = message/star-randomness-response
+content-length = Ne
 
-As we discuss in {{leakage}}, STAR leaks deterministic tags derived from the client measurement that reveal which (and how many) clients share the same measurements, even if the measurements themselves are not revealed. This also enables an online dictionary attack to be launched by the Aggregation Server by sending repeated VOPRF queries to the Randomness Server as discussed in {{dictionary-attacks}}.
+<Bytes containing a serialized evaluated element>
+~~~
 
-The leakage of Prio is defined as whatever is leaked by the function that the aggregation computes. The leakage in Poplar allows the two Aggregation Servers to learn all heavy-hitting prefixes of the eventual heavy-hitting strings that are output. Note that in Poplar it is also possible to launch dictionary attacks of a similar nature to STAR by launching a Sybil attack {{Sybil}} that explicitly injects multiple measurements that share the same prefix into the aggregation. This attack would result in the aggregation process learning more about client inputs that share those prefixes.
+Upon receipt, the client computes `evaluated_element = DeserializeElement(randomness_response)`.
+If this fails, the client aborts the protocol. Otherwise, the client
+finalizes the OPRF protocol to compute the output `rand` as follows:
 
-Finally, note that under collusion, the STAR security model requires the adversary to launch an offline dictionary attack against client measurements. In Prio and Poplar, security is immediately lost when collusion occurs.
+```
+rand = client_context.Finalize(msg, blind, evaluated_element)
+```
 
-## Support for auxiliary data
+## Reporting Phase {#client-message}
 
-It should be noted that clients can send auxiliary data ({{auxiliary-data}}) that is revealed only when the aggregation including their measurement succeeds (i.e. K-1 other clients send the same value). Such data is supported by neither Prio, nor Poplar.
+In the reporting phase, the client uses its measurement `msg` with auxiliary data `aux`
+and its derived randomness `rand` to produce a report for the Aggregation Server.
 
-# Security Considerations {#security-considerations}
+### Reporting Configuration
 
-## Randomness Sampling {#sec-randomness-sampling}
+The reporting phase requires the Aggregation Server to be configured with a URI for
+accepting reports. As an example, the Aggregation Server URI might be https://aggregator.example.
+The Aggregation Server is both an Oblivious HTTP Target and Oblivious Gateway Resource.
```

Comment:
I think we could mention something like "If we use OHTTP", but this seems like we're requiring OHTTP.